### ISSUE TITLE
%autorelease missing %commit

### ISSUE THREAD

**Issue by eclipseo** [juhp/fbrnch#39]
Output:

```
== golang-github-softlayer-xmlrpc f38 ==
Merging from f39:
d54b382 Initial import (2023-08-19)

Press Enter to push and build: 
golang-github-softlayer-xmlrpc-1.0-1.fc38

   061f74d..d54b382  d54b382b507e3fad083d6b708baadb4882407b55 -> f38
14:20:25 Task info: https://koji.fedoraproject.org/koji/taskinfo?taskID=105008252
Watching tasks (this may be safely interrupted)...
105008252 build (f38, /rpms/golang-github-softlayer-xmlrpc.git:d54b382b507e3fad083d6b708baadb4882407b55): free
105008252 build (f38, /rpms/golang-github-softlayer-xmlrpc.git:d54b382b507e3fad083d6b708baadb4882407b55): free -> open (buildvm-x86-31.iad2.fedoraproject.org)
  105008253 buildSRPMFromSCM (/rpms/golang-github-softlayer-xmlrpc.git:d54b382b507e3fad083d6b708baadb4882407b55): open (buildvm-x86-08.iad2.fedoraproject.org)
  105008253 buildSRPMFromSCM (/rpms/golang-github-softlayer-xmlrpc.git:d54b382b507e3fad083d6b708baadb4882407b55): open (buildvm-x86-08.iad2.fedoraproject.org) -> closed
  0 free  1 open  1 done  0 failed
  105008261 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, ppc64le): open (buildvm-ppc64le-03.iad2.fedoraproject.org)
  105008262 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, s390x): free
  105008259 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, x86_64): open (buildvm-x86-20.iad2.fedoraproject.org)
  105008260 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, aarch64): open (buildvm-a64-21.iad2.fedoraproject.org)
  105008262 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, s390x): free -> open (buildvm-s390x-27.s390.fedoraproject.org)
  105008260 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, aarch64): open (buildvm-a64-21.iad2.fedoraproject.org) -> closed
  0 free  4 open  2 done  0 failed
  105008259 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, x86_64): open (buildvm-x86-20.iad2.fedoraproject.org) -> closed
  0 free  3 open  3 done  0 failed
  105008262 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, s390x): open (buildvm-s390x-27.s390.fedoraproject.org) -> closed
  0 free  2 open  4 done  0 failed
  105008261 buildArch (golang-github-softlayer-xmlrpc-1.0-1.20230721git5f089df.fc38.src.rpm, ppc64le): open (buildvm-ppc64le-03.iad2.fedoraproject.org) -> closed
  0 free  1 open  5 done  0 failed
  105008324 tagBuild (noarch): closed
105008252 build (f38, /rpms/golang-github-softlayer-xmlrpc.git:d54b382b507e3fad083d6b708baadb4882407b55): open (buildvm-x86-31.iad2.fedoraproject.org) -> closed
  0 free  0 open  7 done  0 failed

105008252 build (f38, /rpms/golang-github-softlayer-xmlrpc.git:d54b382b507e3fad083d6b708baadb4882407b55) completed successfully
took 5 min 45 sec
14:26:11
review bug: https://bugzilla.redhat.com/show_bug.cgi?id=2224623

Implementation of XMLRPC protocol in Go language for the SoftLayer api

- https://github.com/softlayer/xmlrpc

Press Enter to use above or input notes: 
Creating Bodhi Update for golang-github-softlayer-xmlrpc-1.0-1.fc38:
{"status": "error", "errors": [{"location": "body", "name": "builds", "description": "Build does not exist: golang-github-softlayer-xmlrpc-1.0-1.fc38"}, {"location": "body", "name": "builds", "description": "You may not specify an empty list of builds."}, {"location": "body", "name": "builds", "description": "ACL validation mechanism was unable to determine ACLs."}]}
bodhi submission failed for golang-github-softlayer-xmlrpc-1.0-1.fc38
Press Enter to resubmit to Bodhi:
```

As you can see the commit data from the release tag is not picked up by fbrnch. I am using autorelease/autochangelog, while packaging a specific commit.

Sample SPEC file:

```rpm
# Generated by go2rpm 1.9.0
%bcond_without check
%global debug_package %{nil}

# https://github.com/softlayer/xmlrpc
%global goipath         github.com/softlayer/xmlrpc
Version:                1.0
%global commit          5f089df7cb7e37ba0cba4e7790910af66f0f7b1a

%gometa -f

%global common_description %{expand:
Implementation of XMLRPC protocol in Go language with some changes to interact
with the SoftLayer api.}

%global golicenses      LICENSE
%global godocs          README.md

Name:           %{goname}
Release:        %autorelease
Summary:        Implementation of XMLRPC protocol in Go language for the SoftLayer api

License:        MIT
URL:            %{gourl}
Source:         %{gosource}

%description %{common_description}

%gopkg

%prep
%goprep
%autopatch -p1

%generate_buildrequires
%go_generate_buildrequires

%install
%gopkginstall

%if %{with check}
%check
%gocheck
%endif

%gopkgfiles

%changelog
%autochangelog
```

**Comment by juhp** [comment]
Thank you for reporting this - will try to address it soon

**Comment by juhp** [comment]
I guess I need to support `%commit` I see

**Comment by juhp** [comment]
Any ideas how to get the correct %release ?

**Comment by juhp** [comment]
Nvm, I guess the answer is `fedpkg verrel`, though I was hoping for some alternative...

**Comment by juhp** [comment]
I have a fix for this, which I had already intended to push here - will do so soon

**Comment by juhp** [comment]
Should be fixed in [1.3.3](https://hackage.haskell.org/package/fbrnch-1.3.3)
